test/src/ui/js/test_Game.js

```diff
@@ -1671,28 +1673,101 @@ test("test_Game.createGameMatDieDiv", function(assert) {
     value: 2
   };
 
+  Api.game = { player: { dieBackgroundType: 'symmetric' } };
+
   var dieDivJQuery = Game.createGameMatDieDiv(die, 'player', 'active', true);
   var dieDivProps = BMTestUtils.DOMNodePropArray(dieDivJQuery[0]);
-  var expectedDivProps = [ "DIV", { "class": "die_img" }, [
+  var expectedDivProps = [ "DIV", { "class": "die_img", "style": "background-image: url(images/die/symmetric/d6active.png)" }, [
     [ "SPAN", { "class": "die_overlay die_number_player" }, [ "2" ] ] ]
   ];
   assert.deepEqual(dieDivProps, expectedDivProps, "dieDiv looks correct for clickable active die");
 
-  var dieDivJQuery = Game.createGameMatDieDiv(die, 'opponent', 'active', false);
+  var dieDivJQuery = Game.createGameMatDieDiv(die, 'opponent', 'active', false, 'symmetric');
   var dieDivProps = BMTestUtils.DOMNodePropArray(dieDivJQuery[0]);
-  var expectedDivProps = [ "DIV", { "class": "die_img die_greyed" }, [
+  var expectedDivProps = [ "DIV", { "class": "die_img die_greyed", "style": "background-image: url(images/die/symmetric/d6inactive.png)" }, [
     [ "SPAN", { "class": "die_overlay die_number_opponent" }, [ "2" ] ] ]
   ];
   assert.deepEqual(dieDivProps, expectedDivProps, "dieDiv looks correct for unclickable active die");
 
-  var dieDivJQuery = Game.createGameMatDieDiv(die, 'player', 'captured', false);
+  var dieDivJQuery = Game.createGameMatDieDiv(die, 'player', 'captured', false, 'symmetric');
   var dieDivProps = BMTestUtils.DOMNodePropArray(dieDivJQuery[0]);
-  var expectedDivProps = [ "DIV", { "class": "die_img" }, [
+  var expectedDivProps = [ "DIV", { "class": "die_img die_dead", "style": "background-image: url(images/die/symmetric/d6taken.png)" }, [
     [ "SPAN", { "class": "die_overlay die_number_player" }, [ "\u00A0" + "2" + "\u00A0" ] ] ]
   ];
   assert.deepEqual(dieDivProps, expectedDivProps, "dieDiv looks correct for captured die");
 });
 
+test("test_Game.backgroundImagePath", function(assert) {
+  Api.game = { player: { dieBackgroundType: 'symmetric' } };
```

Comment:
No, Game.backgroundImagePath() doesn't take the background type as an argument. It reads it directly from Api.game.player.dieBackgroundType:

    return 'url(images/die/' +
         Api.game.player.dieBackgroundType +
         '/d' +
         sidesRoundedUp +
         imageType +
         '.png)';

If you think this behaviour is incorrect, let me know at which level you would like the player-selected background type injected into the logic. The higher up it's injected, the more we just passing around a parameter that is only used in this one place.